deep_ep/buffer.py

```diff
@@ -497,7 +497,8 @@ def low_latency_dispatch(self, x: torch.Tensor, topk_idx: torch.Tensor,
 
     # noinspection PyTypeChecker
     def low_latency_combine(self, x: torch.Tensor, topk_idx: torch.Tensor, topk_weights: torch.Tensor,
-                            handle: tuple, async_finish: bool = False, return_recv_hook: bool = False) -> \
+                            handle: tuple, async_finish: bool = False, return_recv_hook: bool = False,
+                            out: torch.Tensor | None = None) -> \
```

Comment:
It is not compatible with Python 3.8, can you change it to `Optional[torch.Tensor]`?